client/src/global.scss

```diff
@@ -183,3 +183,12 @@ input[type='number'] {
   outline: 1px dashed hotpink;
   padding: 0.33em;
 }
+
+.offline-toaster {
+  margin-top: 180px;
+  width: 20%;
+  min-width: 200px;
+  left: 75%;
```

Comment:
Isn't there a way to position the Toast relative to the `ion-content` of the page?
So we can position it "in the top-right"?

Maybe using the `var(--ion-margin)`-value as spacing around it?